### PR TITLE
Closes 89,93,88,86

### PR DESCRIPTION
**Pull request by ana-cc**

### CODE REVIEW

File: draft-ietf-tsvwg-dscp-considerations-09.xml [hunk 1]
```diff
@@ -440,7 +440,7 @@ configured to support DiffServ. (See also <xref target="remark"></xref>).
         </list></t>
 	    
         <t>NOTE: More than one mechanism could result in the
-        same DSCP remarking (see below). The behaviors were measured on Internet paths between 2017 and 2021.
+        same DSCP remarking (see below). The behaviors were measured on both IPv4 and IPv6 Internet paths between 2017 and 2021.
```
Comment (gorryfair) [inline draft-ietf-tsvwg-dscp-considerations-09.xml:443]: I suspect we need more; I think Martin is really looking for whether the behaviour was the same for v4 and v6, or whether some pathologies (!) - patterns of behaviour - were different for v6, e.g. was ToS remarking the same? This seems a fair question. `if we don't know we should at least say something about v4 and v6.

File: draft-ietf-tsvwg-dscp-considerations-09.xml [hunk 2]
```diff
@@ -701,11 +701,10 @@ If such a remarked packet further traverses other Diffserv domains, it would rec
 
         <t></t>
         <t><xref target="RFC8325">RFC8325</xref> notes inconsistencies that
-        can result from such remarking, and recommends how to perform this
-        remarking. It proposes several recommendations for mapping a DSCP to an IEEE 802.11 UP for
-        wired-to-wireless interconnection. The recommendation includes mapping network control traffic CS6 and CS7, as well unassigned DSCPs, to UP 0.</t>
+        can result from such remarking, and recommends a different mapping to perform this
+        remarking. It proposes several recommendations for mapping a DSCP to an IEEE 802.11 UP for interconnection between wired and wireless networks. The recommendation in Section 5.1.2 includes mapping network control traffic CS6 and CS7, as well unassigned DSCPs, to UP 0 in the upstream direction (wireless-to-wired). RFC8325 separately proposes mapping CS6 and CS7 traffic to UP 7, in the downstream direction (Section 4.1). </t>
```
Comment (gorryfair) [inline draft-ietf-tsvwg-dscp-considerations-09.xml:705]: is the comma after DSCPs correct`/

File: draft-ietf-tsvwg-dscp-considerations-09.xml [hunk 3]
```diff
@@ -701,11 +701,10 @@ If such a remarked packet further traverses other Diffserv domains, it would rec
...
 
-        <t>In the upstream direction (wireless-to-wired interconnections, this mapping can result in a specific DSCP remarking behavior.
+        <t>As noted above, RFC8325 also proposes a mapping based on multiplying by 8 in the upstream direction. This mapping can result in a specific DSCP remarking behavior.
```
Comment (gorryfair) [inline draft-ietf-tsvwg-dscp-considerations-09.xml:707]: We should say what is multiplied by 8?

**Review by gorryfair — COMMENTED** [review]
see comments

File: draft-ietf-tsvwg-dscp-considerations-09.xml [hunk 4]
```diff
@@ -701,11 +701,11 @@ If such a remarked packet further traverses other Diffserv domains, it would rec
...
-        remarking. It proposes several recommendations for mapping a DSCP to an IEEE 802.11 UP for
-        wired-to-wireless interconnection. The recommendation includes mapping network control traffic CS6 and CS7, as well unassigned DSCPs, to UP 0.</t>
+        can result from such remarking, and recommends a different mapping to perform this
+        remarking. It proposes several recommendations for mapping a DSCP to an IEEE 802.11 UP for interconnection between wired and wireless networks. The recommendation in Section 5.1.2 includes mapping network control traffic CS6 and CS7, as well unassigned DSCPs, to UP 0 in the upstream direction (wireless-to-wired). RFC8325 separately proposes mapping CS6 and CS7 traffic to UP 7, in the downstream direction (Section 4.1). </t>
 
-        <t>In the upstream direction (wireless-to-wired interconnections, this mapping can result in a specific DSCP remarking behavior.
+        <t>As noted above, RFC8325 also proposes a mapping in the upstream direction that remarks the DSCP to the avlue of the UP multiplied by 8. 
+	This mapping can result in a specific DSCP remarking behavior.
```
Comment (gorryfair) [inline draft-ietf-tsvwg-dscp-considerations-09.xml:708]: Ana please check my rewording

File: draft-ietf-tsvwg-dscp-considerations-09.xml (hunk 1 above)
Comment (ana-cc) [reply draft-ietf-tsvwg-dscp-considerations-09.xml:445]: There was no difference if I recall correctly, will check the paper

**Comment by ana-cc** [comment]
Should read better now, and I've added another sentence for IPv6.

**Review by gorryfair — APPROVED** [review]
Thanks, I approve this, but I made changes - I think you now need a careful read of the changes and resolve any final changed face-to-face.

**Pull request closed without merging**